src/signals/incident/components/form/FileInputRenderer/FileInputRenderer.tsx

```diff
@@ -1,10 +1,20 @@
 // SPDX-License-Identifier: MPL-2.0
```

Comment:
suggestion: maybe we should put __tests/files into the parent ,cd ../, directory?!